### DISCUSSION TITLE
Error on preview -- not working at all

### DISCUSSION THREAD

**Discussion by jramboz** [flowershow/flowershow#536] [General and Help]
I'm trying out Flowershow, and I'm following the tutorial. `npx flowershow@latest preview` runs fine, but when I try to connect I only see this error:

>Server Error
>Error: The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.
>This error happened while generating the page. Any console logs will be displayed in the terminal window.

Screenshot also attached. Any thoughts?

![Screenshot 2023-07-25 at 4 58 20 PM](https://github.com/datopian/flowershow/assets/1568691/30517ea6-af22-463c-a08e-9aea6dbd6563)

**Comment by rufuspollock** [comment]
Hi, i think you have same issue as mentioned in https://github.com/datopian/flowershow/issues/508 - can you take a look and see if it answers your issue.
